index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Inter:wght@700&display=swap" rel="stylesheet"> 
+  <link rel="stylesheet" href="./css/style.css">
+  <title>My Portfolio - Squeleton</title>
+</head>
+<body>
+  <nav class="toolbar">
+    <nav class="welcome">
```

Comment:
- [ ] This is not going to be a navigation menu, per convention, kindly use `<a></a>` instead of nav, for the "Welcome"

Note: please keep the nav tag for the hamburger menu image.